nautobot/core/templatetags/helpers.py

```diff
@@ -858,6 +868,65 @@ def querystring(request, **kwargs):
         return ""
```

Comment:
can we replace the body of `querystring()` with a call to `legacy_querystring()` instead of duplicating the code?